examples/webgl_postprocessing_ssao.html

```diff
@@ -120,24 +122,22 @@
 				// Init gui
 				var gui = new dat.GUI();
 				gui.add( postprocessing, "enabled" );
-				gui.add( postprocessing, "ao_only", false ).onChange( renderModeChange );
-				gui.add( postprocessing, "radius" ).min( 0 ).max( 64 ).onChange( radiusChange );
 
-				window.addEventListener( 'resize', onWindowResize, false );
+				gui.add( postprocessing, "onlyAO", false ).onChange( function( value ) { ssaoPass.onlyAO = value; } );
```

Comment:
It would be great if the JavaScript code does not mix `"` and `'`. We normally use `'`. (This is a general style problem of our code base and not specific to your example).